contracts/src/Zappers/LeverageLSTZapper.sol

```diff
@@ -55,10 +59,14 @@ contract LeverageLSTZapper is GasCompZapper, IFlashLoanReceiver, ILeverageZapper
     function openLeveragedTroveWithRawETH(OpenLeveragedTroveParams calldata _params) external payable {
         require(msg.value == ETH_GAS_COMPENSATION, "LZ: Wrong ETH");
 
+        IERC20 collTokenCached = collToken;
+
+        // Set initial balances to make sure there are not lefovers
+        _setInitialBalances(collTokenCached);
```

Comment:
Couldn't we save the balances in local variables here, and send the leftovers from this function (after the return of `flashLoanProvider.makeFlashLoan`) avoiding the use of storage?

Reply:
Ah, yes, good point!